### ISSUE TITLE
master is not current version

### ISSUE THREAD

**Issue by bfirsh** [segmentio/analytics-go#161]
I just wasted 30 mins reading `analytics.go` trying to figure out why my code wasn't working, until I realised the notice in the readme that the current version is in another branch.

A few potential solutions

- Change the GitHub default branch to `v3.0`
- Make the notice in the readme clearer (it is very easy to skip over at the moment!)
- Do something clever to make master backwards compatible (I am not a go packaging expert, so maybe this is not possible...)

**Comment by bsneed** [comment]
@bfirsh You are totally right.  That is unacceptable.  I'm trying to remedy this internally and will respond back to this with a path forward.

**Comment by sanscontext** [comment]
Hey @bfirsh good to see ya. :) Pushing a quick improvement to the Readme for now, but others are also looking at this.

**Comment by bfirsh** [comment]
@sanscontext ah! hi! :D

Thanks y'all for the speedy response. :)

**Comment by prayansh** [comment]
Hi! @bfirsh we have updated the readme's and also updated the default branch for the repository. Closing this out
